code/game/jobs/job/civilian.dm

```diff
@@ -178,11 +178,13 @@
 	health_modifier = 5
 	outfit_type = /decl/hierarchy/outfit/job/service/janitor
 
-	perks = list(PERK_MARKET_PROF, /datum/perk/job/jingle_jangle, /datum/perk/neat) //Union has revoked their chemistry privileges
+	perks = list(PERK_MARKET_PROF, /datum/perk/job/jingle_jangle, /datum/perk/neat, /datum/perk/sure_step) //Union has revoked their chemistry privileges
```

Comment:
I dont think giving them a backround perk is a good idea for this

Reply:
Okay. Why?

Reply:
does something break if you stack the perks or something? Or are you disagreeing from a balance perspective?

Reply:
I'll just nope it anyways it was the only thing I was kinda iffy about in this PR felt abit to good.

Reply:
please still explain if this is a issue with coding or balance. If its a code issue I'd like to understand what goes wrong